wiki/docs/message/localization.md

```diff
@@ -87,6 +87,7 @@ annotation.
 
 !!! warning the dollar sign ($)
     Please note that the character `$` is forbidden in bundle names.
+    Additionally, the bundle name 'jdac' is reserved, for more information visit [this section](#localization-of-content-used-by-the-framework) 
```

Comment:
This would be the correct link, right?
```suggestion
    Additionally, the bundle name 'jdac' is reserved, for more information visit [this section](#localization-of-default-messages) 
```